f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py

```diff
@@ -56,7 +56,9 @@ def create_listener(self, service, bigips):
             tls['name'] = vip['name']
             tls['partition'] = vip['partition']
 
+        network_id = service['loadbalancer']['network_id']
         for bigip in bigips:
+            self.service_adapter.get_vlan(vip, bigip, network_id)
```

Comment:
This diff only shows new code to explicitly add vlan/tunnel ID to the allowed list for a VS.  How do vlan/tunnel IDs get added for members connected a different network than the listener?  I'd expect to see a similar get_vlan() call for each network ID in service['loadbalancer'['networks'].